tests/unit/src/utils/removeLeadingSlashes.spec.ts

```diff
@@ -0,0 +1,17 @@
+import removeLeadingSlashes from "@/utils/remove-leading-slashes/removeLeadingSlashes";
+
+describe("removeLeadingSlashes()", () => {
+  it("should remove slash in the start", () => {
+    expect(removeLeadingSlashes("/about")).toBe("about");
+  });
+  it("should remove all slashes if there are only slashes", () => {
+    expect(removeLeadingSlashes("////")).toBe("");
+  })
+  it("shouldn't remove last slashes", () => {
```

Comment:
Let's use just `should` and then description of the test case

Reply:
Replaced **"shouldn't remove last slashes"** with **"should keep last slashes"**